<launch>
  <arg name="enable_color"        default="true"/>
  <arg name="color_width"         default="1920"/>
  <arg name="color_height"        default="1080"/>
  <arg name="color_fps"           default="30"/>
  <arg name="color_topic"         default="/realsense/color/image_raw"/>
  <arg name="detect_topic"        default="img_detect"/>
  <arg name="human"               default="false"/>
  <arg name="vis"                 default="true"/>

  <!-- <include file="$(find nicol_rh8d)/launch/realsense.launch">
    <arg name="enable_color"        value="$(arg enable_color)"/>
    <arg name="color_width"         value="$(arg color_width)"/>
    <arg name="color_height"        value="$(arg color_height)"/>
    <arg name="color_fps"           value="$(arg color_fps)"/>
  </include> -->

  <node pkg="nicol_rh8d" name="detector_node" type="keypoint_detect.py" output="screen" required="true">
  	<param name="color_topic"     value="$(arg color_topic)" />
    <param name="detect_topic"    value="$(arg detect_topic)" />
    <param name="human"           value="$(arg human)" />
    <param name="vis"             value="$(arg vis)" />
  </node>

</launch>
